frontend/src/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { Blog } from "../BlogCard/Blog.ts";
+import BlogCard from "../BlogCard/BlogCard.tsx";
+import Header from "../Header/Header.tsx";
+
+
+const Homepage: React.FC = () => {
+    const [blogs, setBlogs] = useState<Blog[]>([]);
+
+    useEffect(() => {
+        fetchBlogs();
+    }, []);
+
+    function fetchBlogs(): void {
+        axios.get("/api/blog").then((response) => {
+            setBlogs(response.data);
+        });
+    }
+
+    function handleCreateBlog(): void {
+
+        console.log("Navigating to create blog page...");
+    }
+// @ts-ignore
+    function handleBlogSaved(): void {
+        fetchBlogs();
+    }
+
```

Comment:
hier solltest du auch was returnen, wenn das blog[] leer ist, da du sonst unten Fehler in der map-methode bekommst beim laden der Blogs, oder wenn das Laden fehlschlägt